tests/test_manifest_evolution.py

```diff
@@ -28,7 +28,10 @@ def examples():
 def test_evolution(input_manifest, component_spec, output_manifest):
     manifest = Manifest(input_manifest)
     component_spec = ComponentSpec(component_spec)
-    evolved_manifest = manifest.evolve(component_spec=component_spec)
+    evolved_manifest = manifest.evolve(
+        component_spec=component_spec,
+        write_run_id=manifest.run_id,
```

Comment:
Can we update this test to provide a different run_id and validate that it's set correctly?